frank/radial_fitters.py

```diff
@@ -745,7 +757,10 @@ def fit(self, u, v, V, weights=1, verbose=False):
                 self._iteration_diagnostics['mean'].append(fit.mean)
 
             count += 1
-
+            
+        if verbose:
+            print()
```

Comment:
I don't need this to print the line at 732, do you?

Reply:
Its there so add a new line after the "    FrankFitter iteration XXX" for the last iteration

Reply:
The print statement uses flush so that it reprints on the same line at each iteration.

Reply:
Yes, but the line following it will begin on the same line as the iteration statement unless you add that print

Reply:
ok